### DISCUSSION TITLE
CARBON GNU/Linux

### DISCUSSION THREAD

**Discussion by firouzian** [NRTC/quick-start-guide#82] [Team Posts]
<div align="right">
‫خوب فعلا تا یک جاهایی توی کربن پیش رفتیم. تقریبا ۷۵٪ ماجرا تمومه. روز رو که مشخص نکردید ولی توی اون روزی که می‌آید یکی دی‌وی‌دی ۱ دبیان ۹٫۸ رو دانلود کنه:‬
</div>

<div align="left">

http://cdimage.debian.org/cdimage/release/current/amd64/iso-dvd/debian-9.8.0-amd64-DVD-1.iso

</div>

**Comment by hodahakim** [comment]
پنجاه و سه درصدش دانلود شده

**Comment by elmiradehghan** [comment]
سلام قطعا به غیر از پنجشنبه روز دیگه ای نمیتونیم بیایم چون اقای حاجی پور نیستن. پس به نظرم قرار شده پنجشنبه. حالا دوستان دقیقشو اطلاع بدن
در مورد دی وی دی یک من که برا دبیان 9 دانلود کردم و فایلشو دارم. دوباره دانلود کنیم؟

**Comment by firouzian** [comment]
@elmiradehghan 

<div align="right">
‫این دی‌وی‌دی شماره یک نسخه ۹٫۸ است، که تازه آمده!‬
</div>
